src/COLVARS/fix_colvars.cpp

```diff
@@ -930,7 +928,7 @@ double FixColvars::compute_scalar()
 /* local memory usage. approximately. */
 double FixColvars::memory_usage()
 {
-  auto bytes = (double) (num_coords * (2*sizeof(int)+3*sizeof(double)));
-  bytes += (double) (nmax*size_one) + sizeof(*this);
+  double bytes = (double) (num_coords * (2*sizeof(int)+3*sizeof(double)));
+  bytes += (double)(double) (nmax*size_one) + sizeof(this);
```

Comment:
`(double)(double)`??

Reply:
Well, you added this in https://github.com/lammps/lammps/commit/e2e6639013c095c520df4a273a424b742a075174

It's perplexing, but I'd rather trust Axel from 4 years ago than myself today :-)

Reply:
I don't. That commit was obviously based on some stream editor magic and this looks like a case where it went it added a typecast for a case that already had one.